crates/ide-assists/src/handlers/from_to_cfg_attr.rs

```diff
@@ -0,0 +1,184 @@
+use ide_db::source_change::SourceChangeBuilder;
+use itertools::Itertools;
+use syntax::{
+    ast::{self, make, AstNode},
+    ted, NodeOrToken, T,
+};
+
+use crate::{AssistContext, AssistId, AssistKind, Assists};
+
+// Assist: from_to_cfg_attr
+//
+// Converts an attribute to a cfg_attr attribute. Converts a cfg_attr attribute to a regular attribute.
```

Comment:
I think a better wording would be `wrap` and `unwrap` instead of convert. (for the comment here and the assist text thats shown to the user)